client/src/containers/PostDashboard.tsx

```diff
@@ -125,7 +150,15 @@ export default function PostDashboard() {
             alignItems: 'center',
           }}
         >
-          <Pagination count={1} color='primary' variant='outlined' />
+          <Pagination
+            count={pageCount}
+            page={page}
+            onChange={(event: React.ChangeEvent<unknown>, pg: number) => {
```

Comment:
```suggestion
            onChange={(event: React.ChangeEvent<{}>, pg: number) => {
```

Reply:
I have to explicitly put unknown or it causes an error, I looked into this on the MUI docs and they also put unknown for this.

``Type 'ChangeEvent<unknown>' is not assignable to type 'ChangeEvent<{}>'.``